test/action_controller/json/include_test.rb

```diff
@@ -36,6 +36,11 @@ def render_without_include
             render json: @author, adapter: :json
           end
 
+          def render_without_associations
+            setup_data
+            render json: @author, adapter: :json, include: []
```

Comment:
I want to remove most of the controller tests are that really now 'SerializableResource' tests.. would clean things up I think